app/assets/stylesheets/partials/_dashboard.scss

```diff
@@ -1,19 +1,17 @@
 .user-info {
   border-top: 10px solid $govuk-blue;
-}
-
-.user-info h2 {
-  margin-top: 1em;
-}
-
-.user-info span {
-  font-weight: bold;
-}
+    h2 {
+      margin-top: 1em;
+    }
+    span {
+      font-weight: bold;
+    }
+  }
 
-.registration-list {
+/*.registration-list {
```

Comment:
If we're no longer using this for anything, can we just delete it? Other than that, LGTM!